### PR TITLE
[WIP] remove clone links

### PR DESCRIPTION
**Pull request by yeli-buonya**
removed 'clone' links from:

/views/survey_assignments/_survey.html.haml

and

/views/rapidfire/question_groups/_question_group.html.haml

Todos:

update routes.rb
survey_controller.rb
any other code associated with cloning

### CODE REVIEW

File: app/views/rapidfire/question_groups/_question_group.html.haml [hunk 1]
```diff
@@ -19,8 +19,6 @@
   .survey__admin-row__col.survey__admin-row__col--actions
     .course-list__col
       = link_to "Edit", edit_question_group_path(question_group)
-    .course-list__col
```
Comment (ragesoss) [inline app/views/rapidfire/question_groups/_question_group.html.haml]: This is the cloning link for a QuestionGroup, not a Survey. This should be kept.

Comment (yeli-buonya) [reply app/views/rapidfire/question_groups/_question_group.html.haml]: Okay, thanks for the update, let me correct that right away

**Comment by ragesoss** [comment]
@yeli-buonya you can check the CI build to see what tests failed. In this case, I think just `survey_admin_spec` is all that was affected.

**Comment by ragesoss** [comment]
Done with #1458

**Pull request closed without merging**